components/rangeSlider/slider.css

```diff
@@ -0,0 +1,135 @@
+.emoji-slider {
+  -webkit-appearance: none;
+  appearance: none;
+  padding: var(--spacing-xs);
+  border: none;
+  width: 100%;
+  outline: none;
+  overflow: hidden;
+}
+
+.emoji-slider::-webkit-slider-thumb:hover {
+  transform: scale(1.2);
+}
+
+.emoji-slider::-moz-range-thumb:hover,
+.emoji-slider::-moz-range-thumb:active {
+  transform: scale(1.2);
+}
+
+.emoji-slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 1.2rem;
+  height: 1.2rem;
+  border-radius: 50%;
+  background: var(--sky-blue);
+  cursor: pointer;
+  margin-top: -0.2rem;
+  border: 2px solid var(--white);
+  box-shadow: var(--slider-shadow);
+}
+
+.emoji-slider::-moz-range-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 1.2rem;
+  height: 1.2rem;
+  border-radius: 50%;
+  background: var(--sky-blue);
+  cursor: pointer;
+  margin-top: -0.2rem;
+  border: 2px solid var(--white);
+}
+
+.emoji-slider::-moz-range-progress {
+  background-color: var(--blue-pr);
+  height: 0.8rem;
+  border-radius: 0.5rem;
+}
+
+.emoji-slider::-webkit-slider-runnable-track {
+  height: 0.8rem;
+  background: var(--blue-pastel);
+  border-radius: 0.5rem;
+}
+
+.emoji-slider::-moz-range-track {
+  height: 0.8rem;
+  background: var(--blue-pastel);
+  border-radius: 0.5rem;
+}
```

Comment:
Same Style has been repeated. You can join class names separated by comma. Individual styles can be written separately.

Reply:
I checked we can't do that for browser specific selectors.

Reply:
yeah i tried the same didn't work,
https://stackoverflow.com/questions/44913155/using-multiple-vendor-specific-css-selectors-at-once
i read from here